packages/plugin-video/src/OdooVideoHtmlDomRenderer.ts

```diff
@@ -16,6 +18,21 @@ export class OdooVideoHtmlDomRenderer extends AbstractRenderer<Node[]> {
             `<div class="media_iframe_video_size">&nbsp;</div>`,
             `<iframe src="${node.src}" frameborder="0" allowfullscreen="allowfullscreen"></iframe>`,
         ].join('');
+        wrapper.querySelector('.css_editable_mode_display').addEventListener('click', () => {
+            this.engine.editor.nextEventMutex(() => {
+                this.engine.editor.execCommand<Core>('setSelection', {
+                    vSelection: {
+                        anchorNode: node,
+                        direction: Direction.FORWARD,
+                    },
+                });
+            });
+        });
+        wrapper.querySelector('.css_editable_mode_display').addEventListener('dblclick', () => {
+            this.engine.editor.nextEventMutex(() => {
+                this.engine.editor.execCommand('openMedia');
+            });
+        });
```

Comment:
Is this also in the `OdooVideo` commit that was merged?

Reply:
I don't think so.